domains/smg.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "JoshMaster404",
+           "email": "josh@halomail.us.to",
+           "discord": "1247715728141058073"
+        },
+    
+        "record": {
+            "CNAME": "x11.x10hosting.com"
```

Comment:
Only says "Apache is working normally", please add more stuff to the website + fix the other issues